magda-storage-api/src/MagdaMinioClient.ts

```diff
@@ -0,0 +1,124 @@
+import { Probe } from "magda-typescript-common/src/express/status";
+import ObjectFromStore from "./ObjectFromStore";
+import ObjectStoreClient from "./ObjectStoreClient";
+import { Stream, Readable } from "stream";
+
+const Minio = require("minio");
```

Comment:
Looks like you can `import` this if you install https://www.npmjs.com/package/@types/minio ?